src/utils/graphql.js

```diff
@@ -1,35 +1,52 @@
-// import { createProvider } from './vue-apollo'
-import ApolloClient from 'apollo-boost'
-import { ApolloLink, concat } from 'apollo-link'
-import axios from 'axios'
-import { buildAxiosFetch } from 'axios-fetch'
+import ApolloClient from 'apollo-client'
+import { ApolloLink, split } from 'apollo-link'
+import { HttpLink } from 'apollo-link-http'
+import { WebSocketLink } from 'apollo-link-ws'
+import { getMainDefinition } from 'apollo-utilities'
+import { InMemoryCache } from 'apollo-cache-inmemory'
 
-export function createApolloClient (uri) {
-  const authMiddleware = new ApolloLink((operation, forward) => {
-    operation.setContext({
-      headers: {
-        // FIXME: this is the random generated password, update every time it is generated for now!
-        Authorization: 'Basic Y3lsYzpiaFRTbnRpWjRqR2YzSWR3eTlVMg=='
```

Comment:
From the old prototype, I suspect the PR for Cylc UI Server auth may require something like this, but we can cross that bridge when time comes, no need to leave an old Cylc Suite passphrase hard-coded here.